test/files/neg/t6810.scala

```diff
@@ -15,14 +15,14 @@ trait t6810 {
   val B = s"""
 """                   // or the same for interpolated strings
 
-  import scala.compat.Platform.EOL
```

Comment:
I would do the import rename here, so the check file stays the same. It's a tricky test.